config/emacs/install/.gitignore

```diff
@@ -1,4 +0,0 @@
-emacs-26
```

Comment:
To make this compatible with arch, let's restore those files. We can move them to another directory or just leave it them in the same path, even if they are linked is not relevant